consensus/src/merkle.rs

```diff
@@ -78,7 +78,7 @@ impl<const H: usize> BinaryMerkle<H> {
 /// Calculate the root of a dynamic merkle tree (containing up to 2^15 elements)
 /// in the same way of how dusk-blockchain does
 ///
-/// For reference impl check here https://github.com/dusk-network/dusk-crypto/blob/master/merkletree/merkletree.go
+/// For reference impl check here <https://github.com/dusk-network/dusk-crypto/blob/master/merkletree/merkletree.go>
```

Comment:
For completeness, on line 110 in this file there's also a link in the tests that could use the angle brackets.

I'm wondering though if it still makes sense to refer to the Go implementation, but that's probably out of scope.